crawl-ref/source/decks.cc

```diff
@@ -1122,7 +1122,8 @@ static void _damaging_card(card_type card, int power,
 
     dist target;
     zap_type ztype = ZAP_DEBUGGING_RAY;
-    const zap_type painzaps[2] = { ZAP_AGONY, ZAP_BOLT_OF_DRAINING };
+    const zap_type painzaps[3] = { ZAP_AGONY, ZAP_BOLT_OF_DRAINING,
+                                   ZAP_BOLT_OF_DRAINING };
```

Comment:
I think just tormenting is a fine Tier 3 effect for pain; no need to zap bolt of draining too.